python/shotgun_fields/image_widget.py

```diff
@@ -312,18 +380,16 @@ def _update_btn_position(self):
 
     def _update_display(self):
         """
-        If no pixmap, display a link to upload if the widget is editable,
-        otherwise display a ``No Image`` string.
+        If no pixmap, display the button with options.
```

Comment:
This is strictly a poorly worded summary of what the method does... fixed and cleaned up the method too.